packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -172,6 +237,9 @@ class Send extends Component {
     } else if (amountBn.isZero()) {
       return { amount: 'Please enter a non-zero amount' };
     } else if (amountBn.isNegative()) {
+      if (this.state.maxSelected) {
```

Comment:
Are you sure this should go here? Here amountBn is negative.

Reply:
When you click "max" with a very low ETH balance, there is no failsafe to keep the amount positive in `calculateMax` [here](https://github.com/paritytech/fether/pull/302/files/bab5b41d64591dfdac4a0da1e14d5166cf1c91d3#diff-c3bd29e19f54d71af4b10fee71e70006R87). This actually allow us to be in this exact state where we know that the user hasn't entered the negative amount himself.

Thinking about it, it's probably too ugly, I'll find some other way to get the info right and the amount positive.